packages/playground/src/components/playground/commits-settings.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Text,
+  Button,
+  DropdownMenu,
+  DropdownMenuTrigger,
+  Icon,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator
+} from '@harnessio/canary'
+import React from 'react'
+
+interface PlaygroundCommitsSettingsProps {
+  loadState: string
+  setLoadState: (state: string) => void
+}
+
+const PlaygroundCommitsSettings: React.FC<PlaygroundCommitsSettingsProps> = ({ loadState, setLoadState }) => {
```

Comment:
these settings components should be in `playground/pages` because they are only for testing in the playground itself.. they will not actually get exported/used in Harness